resources/assets/sass/components/partials.scss

```diff
@@ -0,0 +1,31 @@
+.alert {
+    padding: 15px;
+    border-radius: 5px;
+    color: $colors-light-background;
+}
+
+.alert--danger {
+    background: $colors-alert-danger;
+}
+
+.alert--success {
+    background: $colors-alert-success;
+}
+
+.alert__fa {
```

Comment:
`.alert__fa` can only be used **inside** `.alert`, so the CSS selector would be `.alert .alert__fa`.